cmd/helm3.go

```diff
@@ -120,7 +120,26 @@ func (d *diffCmd) template(isUpgrade bool) ([]byte, error) {
 		flags = append(flags, "--set-string", stringValue)
 	}
 	for _, valueFile := range d.valueFiles {
-		flags = append(flags, "--values", valueFile)
+		if strings.TrimSpace(valueFile) == "-" {
+			var bytes []byte
+			var err error
+
+			bytes, err = ioutil.ReadAll(os.Stdin)
```

Comment:
```suggestion
			bytes, err := ioutil.ReadAll(os.Stdin)
			if err != nil {
				return nil, err
			}
```

Sry I missed it in my first iteration but you seem have missed handling the error and you don't need to separate var declarations as you don't mutate the vars `bytes` and `err` in the scope.